lua/swamp/misc/sh_titles.lua

```diff
@@ -213,3 +276,24 @@ AddTitle("", {
 --NOMINIFY
 --TODO: titles where you don't have a description but the title itself hints at what you do? like a secret achievement but not totally secret
 --TODO: if threshold=true, its 1 but you can put better text on the button in that case
+
+if SERVER then
+    local TitleFunctions =
+    {
+        "weapon_gauntlet" = function(atk) atk:AddStat("snaps") end,
+        "weapon_slitter" = function(atk,vic) if vic:IsPlayer() && not vic:IsAfk() then atk:AddStat("knifekill") end end,
+        "dodgeball" = function(atk,vic) atk:AddStat("dodgeballkill") end,
+        "weapon_fists" = function(atk) atk:AddStat("fistkill") end
+    }
+
+    hook.Add("PlayerDeath", "TitlesPlayerDeath", function(vic,inf,atk)
+        if atk:IsPlayer() then
+            if TitleFuncions[inf:GetClass()] then TitleFunctions[inf:GetClass()](atk,vic) end
+            if vic:IsPlayer() && not vic:IsAfk() && vic:LastHitGroup() == 1 then atk:AddStat("headshotkill") end
+            if vic:GetModel() == "models/player/kleiner.mdl" then atk:AddStat("kleinerkiller") end
```

Comment:
kleinerkill for consistency. once you set this its a hassle to change it

Reply:
Done.